src/test/java/gov/healthit/chpl/aqa/pageObjects/CMSidReverseLookupPage.java

```diff
@@ -56,17 +52,16 @@ public static WebElement generateCmsIdButton(final WebDriver driver) {
      * @return element holding main content
      */
     public static WebElement mainContent(final WebDriver driver) {
-        element = driver.findElement(By.id("main-content"));
-        return element;
+        return driver.findElement(By.id("main-content"));
```

Comment:
I wonder if this `mainContent` element could move to the `BasePage` class. It _should_ be on all pages, or at least, I think it should, based on what I think we've done wrt 508 stuff...